src/components/Search/SearchInputSelectionWrapper/index.native.tsx

```diff
@@ -1,10 +1,51 @@
 import type {ForwardedRef} from 'react';
-import React, {forwardRef} from 'react';
+import React, {forwardRef, useEffect, useState} from 'react';
+import {View} from 'react-native';
+import {Rect} from 'react-native-svg';
 import SearchAutocompleteInput from '@components/Search/SearchAutocompleteInput';
 import type {SearchAutocompleteInputProps} from '@components/Search/SearchAutocompleteInput';
+import ContentLoader from '@components/SkeletonViewContentLoader';
 import type {BaseTextInputRef} from '@components/TextInput/BaseTextInput/types';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
 
+// we need to delay the first render of the SearchAutocompleteInput on native because initialization takes especially long, web doesn't have this issue
```

Comment:
nitpick:
```suggestion
// We need to delay the first render of the SearchAutocompleteInput on native because initialization takes especially long, web doesn't have this issue
```